server/asset/btc/btc.go

```diff
@@ -477,9 +477,10 @@ out:
 			if *bestHash == tip.hash {
 				continue
 			}
+			best := bestHash.String()
 			block, err := btc.node.GetBlockVerbose(bestHash)
 			if err != nil {
-				btc.log.Errorf("error retrieving block %s: %v", bestHash, err)
+				btc.log.Errorf("error retrieving block %s: %v", best, err)
```

Comment:
%v or %s should work with a `Stringer` like `chainhash.Hash`.  Did this print something unexpected?

Reply:
`bestHash` is getting updated in the tests, the read here causes the data race as result.

Reply:
aaaah, it's a pointer, good catch